website/docs/topics/feature-flags/feature-flag-best-practices.mdx

```diff
@@ -85,11 +171,13 @@ For optimal performance, you should evaluate feature flags as close to your user
 
 In summary, this principle emphasizes optimizing performance while protecting end-user privacy by evaluating feature flags as close to the end user as possible. This also leads to a highly available feature flag system that scales with your applications.
 
-## 4. Scale horizontally by decoupling reads and writes
+<VideoContent videoUrls={["https://www.youtube.com/embed/mCXSAWzdn3I"]}/>
```

Comment:
Not a big fan of that video here. I know we don't have a lot of choice, so maybe let's use this as an opportunity to make one? Or ask the team who wrote it to do that

Reply:
Sorry, this is in the wrong place, it's meant to go with the SDKs section, just got tangled up during a merge conflict